planet/subscription_request.py

```diff
@@ -127,6 +127,18 @@ def catalog_source(
         planet.exceptions.ClientError: If start_time or end_time are not valid
             datetimes
     '''
+    if len(item_types) > 1:
+        raise ClientError(
+            """Subscription can only be successfully created if one item type
+            is specified.""")
```

Comment:
```suggestion
        raise ClientError(
            "Subscription can only be successfully created if one item type",
            "is specified.")
```
the triple-quotes introduced an awkward line wrap
```
planet-client-python$> planet subscriptions request-catalog --item-types REScene,PSScene --asset-types basic_analytic_b2 --geometry geometry.geojson --start-time 2022-02-02
Error: Subscription can only be successfully created if one item type
            is specified.
```

Reply:
Nice catch! This is much nicer :)